wkcuber/api/Dataset.py

```diff
@@ -15,6 +16,34 @@
 from wkcuber.api.View import View
 
 
+def is_int(s: str) -> bool:
+    try:
+        int(s)
+        return True
+    except ValueError:
+        return False
+
+
+def dtype_per_layer_to_dtype_per_channel(dtype_per_layer, num_channels):
+    if dtype_per_layer is None:
+        return None
+    # split the dtype into the actual type and the number of bits
+    # example: "uint24" -> ["uint", "24"]
+    dtype_per_layer_parts = re.split("(\d+)", str(dtype_per_layer))
+    # calculate number of bits for dtype_per_channel
+    dtype_per_channel_parts = [
+        (str(int(int(part) / num_channels)) if is_int(part) else part)
+        for part in dtype_per_layer_parts
+    ]
```

Comment:
Let's say someone creates a layer with 3 channels.
If that person would use `"int"` as `dtype_per_layer`, then `dtype_per_channel` would still be `int` and therefore valid. However, the normal `int` has 32 bits, which means that one channel should have `32/3` bits (which does not work). 
Should I simply add a check that the `dtype_per_layer` must have a number in it (e.g. `uint24`) if the layer has multiple channel?

Reply:
> Should I simply add a check that the dtype_per_layer must have a number in it (e.g. uint24) if the layer has multiple channel?

Yes, I think, it is fair to require the dtype to be specific (i.e., also contain a number).